validator/validator.proto

```diff
@@ -238,6 +238,7 @@ message AmpLayout {
     CONTAINER = 5;
     FILL = 6;
     FLEX_ITEM = 7;
+    INTRINSIC = 8;
```

Comment:
let's add validator tests to `validator/testdata/feature_tests/amp_layouts.html` and  `validator/testdata/feature_tests/amp_layouts.out` for the tags that support intrinsic now